### ISSUE TITLE
Error on 0.3 version

### ISSUE THREAD

**Issue by vdeville** [sbrl/soundbox#3]
Hello,
From the last rewrite in 0.3 version. Chrome error

<img width="524" alt="capture d ecran 2018-05-21 a 00 06 37" src="https://user-images.githubusercontent.com/6104498/40284168-51dda504-5c8b-11e8-8fc8-97849793ac6c.png">

**Comment by vdeville** [comment]
Partial fix when specify type=module when script is import, but another error when library is use `SoundBox is not defined`

**Comment by vdeville** [comment]
Sample code
```html
<script rel="script" type="module" src="/js/soundbox.js"></script>
<script>
    let soundbox = new SoundBox();
</script>
```

`Uncaught ReferenceError: SoundBox is not defined`

Do you have the solution ? I'm not skilled with ecma6

Thanks

**Comment by sbrl** [comment]
I see. It looks like you're not using ES6 modules in your project. Check out [the demo](https://sbrl.github.io/soundbox/example.html) - it uses ES6 modules. You've got to make _all_ your code use ES6 modules, or none of it.

Fear not, however - I'll set up 2 different versions. One that uses ES6 modules, and one that doesn't.

**Comment by sbrl** [comment]
It's ready! I've released [v0.3.1](https://github.com/sbrl/soundbox/releases/tag/v0.3.1) that contains a non-es6 module version.

The thing with ES6 modules is that it's all-or-nothing. You can't mix one and the other, because each module file has it's own scope. You've got to use `import` and `export` to shift things around, but things can't be `export`ed outside of the module system, as you've discovered. To use the ES6 module version, you'd need to utilise ES6 modules throughout your codebase.
